locales/en/index.yml

```diff
@@ -1582,11 +1583,24 @@ send_email_messages:
       info: You will not receive email messages anymore
     enable_all:
       label: Enable for all active services
-      info: Every message will be forwarded to your email, if allowed by the service
+      info: Every message will be forwarded to your epreferencesmail, if allowed by the service
     by_service:
       label: Select service by service
       info: You will be able to select from each service settings screen to receive its messages by email as well
 services:
+  optIn:
+    preferences:
+      title: "How do you want to use the services on IO?"
+      body: "IO is simple because you don't have to do anything: if a service has something important to tell you, it can send you a message. Alternatively, you can activate services one by one."
```

Comment:
```suggestion
      body: "IO is simple because there's nothing to set up: if a service has something important to tell you, it can send you a message. Alternatively, you can activate services one by one."
```